inc/class-sitemaps.php

```diff
@@ -267,13 +267,15 @@ function redirect( $query ) {
 
 		if ( $caching ) {
 			do_action( 'wpseo_sitemap_stylesheet_cache_' . $type, $this );
-			$this->sitemap = get_transient( 'wpseo_sitemap_cache_' . $type . '_' . $this->n );
+
+			$sitemap_cache_key = WPSEO_Utils::get_sitemap_cache_key( $type, $this->n );
```

Comment:
I understand the $n variable already existed, but it is unclear what this exactly is. Could you please refactor this to a variable with a more clear name?